build/AzurePipelinesTemplates/WindowsAppSDK-PackNuget-Steps.yml

```diff
@@ -0,0 +1,166 @@
+parameters:
+- name: "SignOutput"
+  type: boolean
+  default: False
+
+steps:
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation x64'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildBinaries_release_x64"
```

Comment:
These are new artifact names for OneBranch, right?
1) Can we simplify the new names like we plan to do for the Aggregator repo?
2) Can we document a mapping between the old artifacts layout and the new?

Reply:
I don't think this intermediatory drop needs to be renamed but I can rename the drop that contains the NugetPackage.

Reply:
I don't think documenting is necessary here. There are no systems downstream that relies on the artifact drop mapping.